speech.js

```diff
@@ -1,5 +1,105 @@
 import fetch from 'isomorphic-fetch';
 import FormData from 'form-data';
+import WebSocket from 'ws';
+
+const DEBUG = false;
+const logDebug = (...args) => {
+  if (DEBUG) {
+    console.debug(...args);
+  }
+};
+
+const URL_STREAMING = 'wss://api.lmnt.com/speech/beta/synthesize_streaming';
+const URL_VOICES = 'https://api.lmnt.com/speech/beta/voices';
+const URL_SYNTHESIZE = 'https://api.lmnt.com/speech/beta/synthesize';
+
+const HEADER_API_KEY = 'X-API-Key';
+
+const MESSAGE_EOF = {"eof": "true"};
+
+class MessageQueue {
+  constructor() {
+    this._messages = [];
+    this._resolvers = [];
+  }
+
+  push(message) {
+    if (this._resolvers.length) {
+      const resolve = this._resolvers.shift();
+      resolve(message);
+    } else {
+      this._messages.push(message);
+    }
+  }
+
+  async getNextMessage() {
```

Comment:
next()?

since it's a message queue and message is already in the name

Reply:
Done.